archived/star-rx72n-firmware/lib/rx_usb/src/rx_usb_cdc.c

```
@@ -742,68 +1176,84 @@ static void internal_handle_set_configuration(const uint16_t usb_value)
+static void internal_handle_set_line_coding(const rx_usb_port_id_t port)
 {
+  if (port >= k_usb_port_count) {
+    usb0()->dcpctr |= k_usb_dcpctr_pid_stall;
+    return;
+  }
+
   uint8_t data[k_line_coding_size];
 
   /* Read line coding data from FIFO */
-  const uint32_t len = rx_usb_hw_fifo_read(k_usb_pipe_0, data, k_line_coding_size);
+  const uint32_t len = rx_usb_hw_fifo_read(k_usb_pipe_dcp, data, k_line_coding_size);
```

Clang (14.0.6)

[warning] variable 'len' is not initialized

(cppcoreguidelines-init-variables)